app.json

```diff
@@ -1,7 +1,7 @@
 {
     "runtime": {
-        "arguments": "--v=1 --inspect",
-        "version": "canary"
+        "arguments": "",
+        "version": "alpha"
```

Comment:
should we keep on canary? Its a good source for finding out issues / regressions...

Reply:
Asking Nicholas as it might be disruptive for customers as we will break stuff.

Reply:
Should we make a develop branch?

Reply:
Also we'll run into this with styling changes, because all my styles would land in canary.

Reply:
Made a dev branch: https://github.com/openfin/platform-api-project-seed/pull/13